src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -292,12 +292,7 @@ def lookup_job_info(self, job_ids: List[str]) -> dict:
                 "job_id": job_id,
                 "job_params": job.params,
             }
-        for error_id in error_ids:
-            infos[error_id] = {
-                "job_id": error_id,
-                "error": DOES_NOT_EXIST,
-            }
-        return infos
+        return self.add_errors_to_results(infos, error_ids)
```

Comment:
uniform handling of the 'job not found' error messages, regardless of what the request is (job status, info, retries, etc.)